.vscode/launch.json

```diff
@@ -4,6 +4,38 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Celery Specific Worker Queue",
+            "type": "python",
+            "request": "launch",
+            "module": "celery",
+            "console": "integratedTerminal",
+            "python": "${workspaceFolder}/env/bin/python",
+            "cwd": "${workspaceFolder}",
+            "args": [
+                "-A",
+                "posthog",
+                "worker",
+                "-B",
+                "--scheduler",
+                "redbeat.RedBeatScheduler",
+                "--without-heartbeat",
+                "--without-gossip",
+                "--without-mingle",
+                "-Q",
+                "CALCULATE_EVENT_PROPERTY_QUEUE"
```

Comment:
If -q specific queues isn't set then the worker reads from the default queue (which is "celery" for us)

A worker can read from more than one queue. So we could have the "throttled queues" workers and the "all the other things" workers

Reply:
This is the point where I don't know how to convince helm to do my bidding

The ideal starting point is to make two sets of exactly the same images 

1) as it is today. process the default queue scale to up to 20 pods (are they pods?)
2) the same thing but started with the -q arg

As it is this code only supports a single queue because it uses the same environment variable. But that's way better than the current situation. Where time -ritical tasks like sending emails are bundled in with things like updating eventually consistent counts and the second can block the first.